docs/generated/packages/angular.json

```diff
@@ -546,9 +549,9 @@
         },
         "additionalProperties": false,
         "required": [
-          "projectPath",
+          "project",
           "componentName",
-          "componentPath",
+          "componentDir",
```

Comment:
for got to update these after switching to project vs paths in initial PR. 
also took it as a change to update the schema to have better props (includes changes to react for to make sure they have the same prompts)